README.md

```diff
@@ -58,38 +58,77 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 
 
-## How to Build CoatRack Container Images from Source
+## Usage
+
+To start using CoatRack: 
+
+- either run CoatRack locally as explained in the "Install ..." sections
+- or open the public instance available at https://coatrack.eu.
+
+To log-in to CoatRack, a Github account is required. After logging in and filling in the registration form, the CoatRack Web Application will open. Use the "Getting started tutorials" that are accessible from inside the application to learn the basics of using CoatRack.
 
- The following prerequisites are required:
+There are two tutorials inside the application:
+
+- Offering service APIs via CoatRack
+- Using service APIs offered via CoatRack
+
+
+
+## Install CoatRack Locally via CLI
```

Comment:
As far as I understand the following instructions, they are to build from source and run locally. Therefore the word "Install" seems misleading to me. I would suggest to replace it by "Build and Run".

Reply:
Done.